docs/README.md

```diff
@@ -5,3 +5,15 @@ The markdown files contained in this path provide documentation for contributing
     pandoc faq.md
 
 Copy the output, excluding the header on the first line, to the source of the relevant page within the CMS.  Note that some of the pages are dynamically generated and are not generated from templates contained here.
+
+# Translation of the FAQ
+
+#### Do you want to translate the FAQ to your language?
```

Comment:
We should generalize this, so that it is not specific to the FAQ.  Other docs include `development.md`, `rulesets.md`